lesson2.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>lesson2</title>
+    </head>
+    <body>
+    </body>
+    <script>
+
+// Задание 1.
+    // var a = 1, b = 1, c, d;
+    // c = ++a; alert(c);           // 2 - префиксный инкремент, сначала сложение, потом использование в выражении
+    // d = b++; alert(d);           // 1 - постфиксный инкремент, сложение после исп. выражения
+    // c = (2+ ++a); alert(c);      // 5 - также, как и в первом случае, a=2 была при первом инкрементировании, до выполнения выражения
+    //                              // прибавляется еще 1, след. c = 5
+    // d = (2+ b++); alert(d);      // 4 - снова постфиксный инкремент, b = 2, после выражения(2) 
+    // alert(a);                    // 3 - т.к. после инкрементирования стояли выражения, то обе переменный в этом месте будут равны 3.
+    // alert(b);                    // 3
+
+// Задание 2.
+    // var a = 2;
+    // var x = 1 + (a *= 2); // как в математике, сначала выполняются выражения в скобках, где a = 4 (префикс)
+
+// Задание 3.
+    function task3(){
+        var a = +prompt('Задайте значение а: ');
+        var b = +prompt('Задайте значение b: ');
+        if (a >= 0 && b >= 0){
+            alert(a - b);
+        } else if(a < 0 && b < 0){
+            alert(a * b);
+        } else if ((a < 0 && b >= 0) || (a >= 0 && b < 0)){
+            alert(a+b);
```

Comment:
Лучше ставить пробелы между операторами и операндами для лучшей читаемости
```suggestion
            alert(a + b);
```